src/test/java/com/github/fppt/jedismock/comparisontests/transactions/TestWatchUnwatch.java

```diff
@@ -156,8 +155,8 @@ public void testWatchWithNoKeyInStorage(Jedis jedis) {
         Transaction transaction = jedis.multi();
         transaction.set(FIRST_KEY, SECOND_VALUE);
         List<Object> result = transaction.exec();
-        assertEquals(1, result.size());
-        assertEquals("OK", result.get(0));
+        assertThat(result).hasSize(1)
```

Comment:
`containsExactly` is sufficient here, no need to check for size

Reply:
Fixed